code/procs/mob_procs.dm

```diff
@@ -924,18 +924,18 @@
 			for (var/datum/mind/M in HR)
 				if (M.current)
 					if (!see_everything && isobserver(M.current)) continue
-					var/I = image(antag_revhead, loc = M.current)
+					var/I = image(antag_revhead, loc = M.current, icon_state = null, layer = 11) //secHuds are on layer 10.
```

Comment:
Can we get these as defines somewhere?

Reply:
Noting the following comment in layer.dm:
```
// Some effects were defined on layer 10, some on layer 20... Lets unify this...
// These are for effects that should display below lighting
```
I've put the icons on EFFECTS_LAYER_UNDER_4 & EFFECTS_LAYER_UNDER_4 +0.1 (trees are generally on under_3). 

Tested in game it looks fine with no obvious standouts.